### ISSUE TITLE
Better handling of non-string argument

### ISSUE THREAD

**Issue by perry-autofi** [tyxla/remove-accents#24]
Would like to be able to send any argument to removeAccents() and have more robust handling. I propose to simply return the sent in value if it's not a string. For example if I pass in undefined simply return undefined. If I pass in an object simply return the object. Don't attempt to call .toString().

**Comment by tyxla** [comment]
Thanks for the suggestions @perry-autofi!

I disagree with this - it would break the predictability, as the library should not be used with non-strings in the first place. Any type checks should be done before running the accent removal function. There's no good reason to call the function with an object or any other non-string value, so I'm closing this as a wontfix.
